RNTester/Podfile.lock

```diff
@@ -501,31 +501,31 @@ SPEC CHECKSUMS:
-  RCTTypeSafety: 1ade47a69b092cddf1e4ea21e0c5bdc65cc950b4
-  React: cafb3c2321f7df55ce90dbf29d513799a79e4418
-  React-callinvoker: 0dada022d38b73e6e15b33e2a96476153f79bbf6
-  React-Core: 6c6f6c40bb1e031de8a0fafce08c010edfef09ab
-  React-CoreModules: d13d148c851af5780f864be74bc2165140923dc7
-  React-cxxreact: 4661b3295e62c6eaada084e2f826c70c71ef11ea
-  React-jsi: fe94132da767bfc4801968c2a12abae43e9a833e
-  React-jsiexecutor: 959bb48c75a3bfc1b1d2b991087a6d8df721cbcf
-  React-jsinspector: 7fbf9b42b58b02943a0d89b0ba9fff0070f2de98
-  React-perflogger: 1f668f3e4d1adef1fafb3b95e7d6cf922113fe31
-  React-RCTActionSheet: 51c43beeb74ef41189e87fe9823e53ebf6210359
-  React-RCTAnimation: 9d09196c641c1ebfef3a4e9ae670bcda5fadb420
-  React-RCTBlob: 715489626cf44d28ee51e5277a4d559167351696
-  React-RCTImage: 19151d2d071b05c3832a0b212473cafa4ea8948f
-  React-RCTLinking: 7c94c0f2fcc658cb4043dacb4f6621dca2f8f8b5
-  React-RCTNetwork: 7596e84acacd5d0674e9743b55c5bf61a626af69
-  React-RCTPushNotification: 88c9f47ff0d4391d5136d70745f15713cdc5f6bb
-  React-RCTSettings: a29c61f85f535ba2eff54d80bef2ea3cdb6e5fba
-  React-RCTTest: cfe25fcf70b04a747dba4326105db398250caa9a
-  React-RCTText: 6c01963d3e562109f5548262b09b1b2bc260dd60
-  React-RCTVibration: d42d73dafd9f63cf758656ee743aa80c566798ff
-  React-runtimeexecutor: 60dd6204a13f68a1aa1118870edcc604a791df2b
-  ReactCommon: 511b4a9ea129c129c6dbc982942007d195903a9a
-  Yoga: f7fa200d8c49f97b54c9421079e781fb900b5cae
+  RCTRequired: 739f267334332bd23fcbdc6105a8ec9311619555
+  RCTTypeSafety: e6821310f3d452d0b25bf33c0c23d8ed94652b60
+  React: ad83d2a55f4b8ccf0a62203049f59f2e98264609
+  React-callinvoker: 41e0b0f92e02ccc32afa03d08d27b1c536a9ecf1
+  React-Core: d92a8b369c4d48975fd7951e7427b66e36ca249e
+  React-CoreModules: eb4c419c09eeabf916fd58e50580454cf47c37a6
+  React-cxxreact: ffa4328d4ac424539c3225acb388c240fe954106
+  React-jsi: c6bf17c1ebf671d2c2c02763ed97cb8c5b3904bc
+  React-jsiexecutor: 545213f625cd58ff93047fb8277af76f696d5488
+  React-jsinspector: b2886388ec5c5eb25a99d0591851328e03c92a67
+  React-perflogger: 1fc65b25c3c9e8afde2b360c310d2eff06bc01de
+  React-RCTActionSheet: a609bc6c6d80a0d94d38b224ec949c786a71c64b
+  React-RCTAnimation: 78815000233aeee7dca6a7890ab4ddb75612a91e
+  React-RCTBlob: 3e26dd1ab750ec1607367aa3e29050e4d85e42b2
+  React-RCTImage: 43321ad54e127cd8c38864cb673e952352805265
+  React-RCTLinking: bc76da619cb7ce6a81e87118f4d952751a19c71c
+  React-RCTNetwork: d7de48eef44f09a71b501eabf7ffbdd65dcf1fee
+  React-RCTPushNotification: 204a642ed98ad336f97b524022715849d65148c9
+  React-RCTSettings: 2d7a13e63fb9b3cb4cbcbd527442af7b3f142c72
+  React-RCTTest: e40fb7d9a3762feba08f25b7dccf0bc2de8370b4
+  React-RCTText: fecc00cae8427e5869445f09c85f33efe0a3f182
+  React-RCTVibration: c6bac747411359e5ee7e590b2dc8939e91c67b5a
+  React-runtimeexecutor: 533b76c7a01b705c651b57838439c5ba96fea654
+  ReactCommon: f17474a4663f81d00462cbe9c944d63fcae3e708
+  Yoga: a98571eb8a4bf47f31675980b61342f9f16b82fe
```

Comment:
All of these changes came if after running `bundle exec pod install` on `master`.

Reply:
Hmm, can you check when it was last updated and what the difference between then and now are f these podspecs?

Reply:
So it looks like all of these specs are local (e.g `FBLazyVector (from \`../Libraries/FBLazyVector\`)` so it would make sense that any change in these directories would cause the checksum to be different. I don't think this is a necessary change in this PR. I'll just remove it.